[FIX]: 빌드 타입 오류 수정

## 수정 사항

```js
src/category/api/delete.ts(17,7): error TS2559: Type '(params: any) => string' has no properties in common with type 'ParamsSerializerOptions'.
```

- [x] axios paramsSerializer에 대한 type 오류 수정

--- a/src/bookmarks/api/bookmark.ts
+++ b/src/bookmarks/api/bookmark.ts
@@ -124,10 +124,14 @@ const DELETEBookMark = {
       params: {
         bookmarkId: params.bookmarkIds,
       },
-      paramsSerializer: (params) => {
-        return Object.keys(params)
-          .map((key) => params[key].map((v: string) => `${key}=${v}`).join('&'))
-          .join('&');
+      paramsSerializer: {
+        encode: (params) => {
+          return Object.keys(params)
+            .map((key) =>
+              params[key].map((v: string) => `${key}=${v}`).join('&'),
+            )
+            .join('&');
+        },
       },
     });
     return data;

--- a/src/category/api/delete.ts
+++ b/src/category/api/delete.ts
@@ -14,10 +14,14 @@ const DeleteCategory = {
       params: {
         categoryId: params.categoryId,
       },
-      paramsSerializer: (params) => {
-        return Object.keys(params)
-          .map((key) => params[key].map((v: string) => `${key}=${v}`).join('&'))
-          .join('&');
+      paramsSerializer: {
+        encode: (params) => {
+          return Object.keys(params)
+            .map((key) =>
+              params[key].map((v: string) => `${key}=${v}`).join('&'),
+            )
+            .join('&');
+        },
       },
     });
     return data;

